zuul.d/project-templates.yaml

```diff
@@ -773,6 +773,19 @@
         - ansible-security-integration-splunk-es-python36
         - ansible-test-sanity-splunk
 
+- project-template:
+    name: ansible-collections-security-trendmicro
+    check:
+      jobs:
+        - build-ansible-collection
+        - ansible-test-security-integration-trendmicro-python38
```

Comment:
We should default to python36 right now.

Reply:
Lets move this job into a 2nd PR, so we can merge the ansible-zuul-jobs changes only. Then once merged, create the new PR for this job, and ask summitt create a integration test playbook to confirm it works properly